src/main/java/org/dependencytrack/model/Bom.java

```diff
@@ -102,6 +102,10 @@ public String getFormatLongName() {
     @NotNull
     private UUID uuid;
 
+    @Persistent
+    @Column(name = "BOM_GENERATED")
+    private Date bomGenerated;
```

Comment:
We should omit the `bom` / `BOM_` prefix here, as the field is already in the `Bom` class, and the column is in the `BOM` table.